cpp/lib/ome/xml/meta/Convert.cpp

```diff
@@ -92,7 +92,6 @@ namespace
       convertFileAnnotations();
       convertListAnnotations();
       convertLongAnnotations();
-      convertMapAnnotations();
```

Comment:
Note that these changes are checked by ./cpp/test/ome-xml/metadata-convert-coverage which will flag up any missing calls.